src/nvim/message.c

```diff
@@ -574,6 +581,10 @@ static bool emsg_multiline(const char *s, bool multiline)
           xfree(p);
         }
         redir_write(s, strlen(s));
+
+        if (State & CMDLINE) {
```

Comment:
Same question as below, but is anything put on screen in this branch (redir write) at all?

Reply:
You are right, this `redrawcmdline` is unnecessary, will be removing it.